### ISSUE TITLE
Difference between tensorflow and pytorch in concatenation

### ISSUE THREAD

**Issue by vinid** [dallascard/scholar#7]
Hello @dallascard,

thanks for the great work!

Quick question: it is not clear to me what you are concatenating in the [TF](https://github.com/dallascard/scholar/blob/master/scholar_tf.py#L203) version and in the [PT](https://github.com/dallascard/scholar/blob/master/scholar.py#L380) version of the code. 

In TF I see you combine labels, covariates and X (as I can also see from eq. 5 in the paper), but in PT I see the combination between PC, TC, and Y with the X. It is not too clear to me what PC and TC are and which is the difference between the code in PT and the TF version (maybe I am just missing something).

Thanks a lot :) :)

**Comment by dallascard** [comment]
Hi @vinid,
Yes, you are correct about that. The TF version is the one that matches the paper. The PT version is somewhat more of a sandbox. Specifically, I was experimenting with incorporating both covariates as topic-like deviations (as described in the paper, and which I call TC in the PT version, for topic covariates), as well incorporating additional observed variables into the prior (which I called PC, for prior covariates), as in Dirichlet Multinomial Regression (https://mimno.infosci.cornell.edu/papers/dmr-uai.pdf). Unfortunately none of this is documented, and I haven't done extensive experiments to figure out what is best to include in the inference. However, unless you specify something using the --prior-covars option, then I believe it should have no effect. Sorry this is not more clear, but at this point this repo remains more of a reference for reproducibility and inspiration than a fully developed implementation.

**Comment by vinid** [comment]
Hello @dallascard,

No problem at all! and thank you for taking the time to explain this! I was just confused about the mismatch, but now that I know what those two terms refer to I am good to go! Thank you again :) :)
